src/utils/utils_windows_common.c

```diff
@@ -0,0 +1,24 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#include <Windows.h>
+#include <processenv.h>
+
+#define BUFFER_SIZE 1024
+
+// Check if the environment variable contains the given string.
+int util_env_var_has_str(const char *envvar, const char *str) {
+    char buffer[BUFFER_SIZE];
+    int rc = GetEnvironmentVariableA(envvar, buffer, BUFFER_SIZE);
+    if (0 < rc && rc < BUFFER_SIZE) {
+        return strstr(buffer, str) ? 1 : 0;
```

Comment:
Please add a function "util_env_var" that returns the value of given var. The util_env_var_has_str shoud use it

Reply:
Done